conans/test/functional/test_profile_detect_api.py

```diff
@@ -18,7 +18,6 @@ def test_profile_detect_compiler(self):
             compiler={{compiler}}
             compiler.version={{detect_api.default_compiler_version(compiler, version)}}
             compiler.runtime={{runtime}}
-            compiler.libcxx={{detect_api.detect_libcxx(compiler, version, compiler_exe)}}
```

Comment:
Why this was not failing before in CI? I thought CI was running VS 17 already

Reply:
Investigating, curious nontheless